ManatiCR/FDI
# FDI-12: Create espacio libre de discrimicacion content type.

[FDI-12](https://manati.atlassian.net/browse/FDI-12)
-
**Steps to test**
- Run `ahoy drush cim -y`
- Log in as an admministrador de contenido user
- Create one node espacio libre de discriminación, check you can edit it and delete it as well
